### ISSUE TITLE
Failed Access-Control-Allow-Methods check while headers are right

### ISSUE THREAD

**Issue by ysv** [rswarthout/payid-validator#17]
**Describe the bug**
Access-Control-Allow-Methods header check marked as failed while headers are right according to validator rules:
![image](https://user-images.githubusercontent.com/19859122/99713570-27c71780-2aad-11eb-8e25-233d973c5f71.png)

![image](https://user-images.githubusercontent.com/19859122/99713745-59d87980-2aad-11eb-9e26-86a301ddfeac.png)

![image](https://user-images.githubusercontent.com/19859122/99713974-aae86d80-2aad-11eb-920d-a05b723581af.png)

**To Reproduce**
PayID: `IDAD45C02BE6$payid.test.coinselect.com`
Network (isn't important): e.g. xrpl-testnet

**Expected behavior**
Mark Access-Control-Allow-Methods check as successful.

**Comment by ysv** [comment]
**Update**

The same behaviour for `Access-Control-Allow-Headers`
![image](https://user-images.githubusercontent.com/19859122/99715977-2cd99600-2ab0-11eb-8883-ef3cdd9ddd07.png)

![image](https://user-images.githubusercontent.com/19859122/99716022-41b62980-2ab0-11eb-9d39-f235e4bb275d.png)

**Comment by rswarthout** [comment]
Yes, this is a bit confusing. The detection for the Access-Control-* headers is done via an OPTIONS preflight request outside the default GET request done to retrieve the PayID details.

<img width="1219" alt="options-request" src="https://user-images.githubusercontent.com/135386/99717899-0566d700-2a78-11eb-95a0-e775eb6c16e2.png">

**Comment by rswarthout** [comment]
@ysv I am checking in to see if you have resolved the issue on your side or if you still believe there is an issue with the validator?

**Comment by ysv** [comment]
@rswarthout 
Currently the issue in on our side.

But I still think that it would be nice if you show OPTIONS request headers in 'Response Headers' or smth like this. So anyone will not get confused like me.

If you think there is no issue on your side please close this github issue. Thanks!

**Comment by rswarthout** [comment]
@ysv That is a fair request. I will work on adding this to the UI. A quick note, it will only be added to paystringvalidator.com as I have frozen payidvalidator.com and will be shutting it down on 2020-12-15.
